modules/planning/common/reference_line_info.cc

```diff
@@ -790,14 +790,15 @@ void ReferenceLineInfo::ExportEngageAdvice(EngageAdvice* engage_advice) const {
   if (!prev_advice->has_advice()) {
     prev_advice->set_advice(EngageAdvice::DISALLOW_ENGAGE);
   }
-  if (!IsDrivable()) {
+
+  if (!IsDrivable() && !FLAGS_enable_start_auto_from_off_lane) {
```

Comment:
"Drivable" can set to FALSE for various reasons. Seems not safe to simply override it with a gflag "start_auto_from_off_lane"

Reply:
Fixed. Thank you!